config/config_test.go

```diff
@@ -73,8 +73,14 @@ func assertFiles(t *testing.T, dir string, files []string) {
 	assert := assert.New(t)
 
 	cwd, _ := os.Getwd()
-	os.Chdir(dir)
-	defer os.Chdir(cwd)
+	if err := os.Chdir(dir); err != nil {
+		t.Error(err)
```

Comment:
This is probably worth `panic`ing on to fail fast; it's not a problem with the code, it's a problem with the test or the underlying os